### DISCUSSION TITLE
“AMD Power Slider” in power plan options

### DISCUSSION THREAD

**Discussion by NatsuZoraKanata** [seerge/g-helper#2427] [General]
Hey everyone in the community! So, I stumbled upon this hidden gem in the Windows power plan settings – it's called the AMD Power Slider. Any idea what it does? Also, is there a way to sync it up with the performance modes?

**Comment by seerge** [comment]
@NatsuZoraKanata quick googling leads me to https://www.reddit.com/r/Amd/comments/uj8ljr/amd_power_slider_in_the_power_plan_options/ :)

Most probably it has no real measurable effect on anything

**Reply by NatsuZoraKanata** [reply]
Well, it seems there's no clear answer about this option on Google. Next, I'll try to test its impact on performance and battery life.

**Reply by NotHIMButHIM** [reply]
Have got any results on this?

**Comment by Redroadsl** [comment]
On my platform(Windows Server 2025 AMD 7800X3D), It has no effect on cpu freq...
